mailscanner/sf_version.php

```diff
@@ -58,31 +58,41 @@
     }
 
     echo '<table width="100%" class="boxtable">' . "\n";
+    echo '<tr><th>Software Versions</th></tr>' . "\n";
     echo '<tr>' . "\n";
     echo '<td>' . "\n";
 
-    echo '<p class="center" style="font-size:20px"><b>' . __('softver11') . '</b></p>' . "\n";
-    echo 'MailWatch ' . __('version11') . ' = ' . $mailwatch_version . '<br>' . "\n";
     echo '<br>' . "\n";
-    echo 'MailScanner ' . __('version11') . ' = ' . $mailscanner_version . '<br>' . "\n";
+    echo 'MailWatch version: ' . $mailwatch_version . '<br>' . "\n";
     echo '<br>' . "\n";
-    echo 'PHP ' . __('version11') . ' = ' . $php_version . '<br>' . "\n";
+    echo 'MailScanner ' . __('version11') . $mailscanner_version . '<br>' . "\n";
+    echo '<br>';
+    echo 'ClamAV ' . __('version11');
```

Comment:
Not everyone uses clamav, there should be a check here (and also there should be no antivirus discrimination ;) )